sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -108,6 +108,43 @@ def test_rfe():
     assert_array_almost_equal(X_r, X_r_sparse.toarray())
 
 
+def test_rfe_sample_weights():
+    iris = load_iris()
+    X = iris.data
+    y = iris.target
+
+    clf = SVC(kernel="linear")
+    rfe = RFE(estimator=clf, n_features_to_select=1)
+
+    sample_weight_test = 2
+    class_test = 2
```

Comment:
I would not use the term `test`. It is kinda confusing.

```python
class_targeted, class_weight_factor = 2, 2
```

Reply:
huh, good point